SharedPackages/DataBrokerProtectionCore/Sources/DataBrokerProtectionCore/Operations/SubJob/SubJobWebRunner/SubJobWebRunner.swift

```diff
@@ -292,6 +301,15 @@ public extension SubJobWebRunning {
     }
 
     func onError(error: Error) async {
+        if let currentAction = actionsHandler?.currentAction(), currentAction is ConditionAction {
+            if actionsHandler?.stepType == .optOut {
+                stageCalculator.fireOptOutConditionNotFound()
+            }
+
+            await executeNextStep()
+            return
+        }
```

Comment:
We probably need to change `retriesCountOnError` for conditional action to 0, otherwise it’ll take the default value of 3 for opt-outs.